Source/Cryptlex.Net/Core/Services/AccountsService.cs

```diff
@@ -17,9 +17,9 @@ public interface IAccountsService :
     {
         Task<AccountLoginResponse> Login(AccountLoginData data);
         Task<AccountLoginGoogleResponse> LoginGoogle(AccountLoginGoogleData data);
-        Task<bool> CheckSSOEnabled(string companyId);
-        Task<Uri> InitiateSSOLogin(string companyId, string returnUrl);
-        Task<Uri> GetSSOReturnUrl(string companyId);
+        Task<bool> CheckSSOEnabled(string CompanyId);
+        Task<Uri> InitiateSSOLogin(string CompanyId, string ReturnUrl);
```

Comment:
As per Microsoft C# coding conventions https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions 
Property names are PascalCase, while parameters and internal variables are camelCase

```csharp
Task<Uri> InitiateSSOLogin(string companyId, string returnUrl);
```